development/dev-image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM nutsfoundation/nuts-node:master as nutsnode
+FROM debian:buster-slim
+
+RUN apt-get update && apt-get install -y curl sudo musl
+RUN curl -sL https://aka.ms/DevTunnelCliInstall | bash
+
+COPY entrypoint.sh /
+RUN chmod +x /entrypoint.sh
+
+COPY --from=nutsnode /usr/bin/nuts /usr/bin/nuts
+RUN chmod +x /usr/bin/nuts
+
+HEALTHCHECK --start-period=30s --timeout=5s --interval=10s \
```

Comment:
do we need a longer starting period because it blocks until you logged in on Github?